apps/data-scanner/lib/models/phones/Phones.mjs

```diff
@@ -0,0 +1,29 @@
+import ModelBase from '../ModelBase.mjs';
+
+export class Phones extends ModelBase {
+    /**
+     * @typedef {Class} Phones
+     * @property getList
+     * @property updateSession
+     */
+
+    /**
+     * @method
+     * @param {String} tracked - flag for phone numbers which should be added to traking
+     * @param {String} withSession - flag for phone numbers which have session field filled
+     * @returns {Promise<Object>}
+     */
+    async getList({ tracked, withSession }) {
+        return await this.repository.phones.read({ tracked, withSession });
+    }
+
+    /**
+     * @method
+     * @param {String} trackedPhone - tracked phonenumber which session we should update
+     * @param {String} sessionId - session identifier
+     * @returns {Promise<Object>}
+     */
+    async updateSession(params) {
```

Comment:
updateSession ---> update

Reply:
const rawPhone = await this.repository.phones.readByPhone(phone);
rawPhone.sessionId = params.sessionId;
return this.repository.phones.update(params);

Reply:
done